packages/agent-utils/src/agent/AgentVariables.ts

```diff
@@ -5,7 +5,7 @@ export class AgentVariables {
   public static Prefix = "${";
   public static Suffix = "}";
 
-  constructor(private _saveThreshold: number = 300) { }
+  constructor(private _saveThreshold: number = 500) { }
```

Comment:
3000 was an experimental number we found was good in practice

Reply:
Okay so there are 2 thresholds at play:
1. The "save threshold" shown here is used to determine what values should be stored as variables. Anything under this threshold is simply printed to chat as a result.
2. The "read variable threshold" which is the maximum amount of tokens you can read when calling "readVariable". This is what I think you're referring to:
https://github.com/polywrap/evo.ninja/blob/a6493637515ff8a4a1c2d7128ed96a60a2f7e7f7/packages/agents/src/functions/ReadVariable.ts#L13